skll/learner/voting.py

```diff
@@ -110,22 +120,22 @@ class VotingLearner(object):
 
     def __init__(
         self,
-        learner_names,
-        voting="hard",
-        custom_learner_path=None,
-        feature_scaling="none",
-        pos_label=None,
-        min_feature_count=1,
-        model_kwargs_list=None,
-        sampler_list=None,
-        sampler_kwargs_list=None,
-        logger=None,
+        learner_names: List[str],
+        voting: Optional[str] = "hard",
+        custom_learner_path: Optional[PathOrStr] = None,
+        feature_scaling: str = "none",
+        pos_label: Optional[LabelType] = None,
+        min_feature_count: int = 1,
+        model_kwargs_list: Optional[List[Dict[str, Any]]] = None,
+        sampler_list: Optional[List[str]] = None,
+        sampler_kwargs_list: Optional[List[Dict[str, Any]]] = None,
+        logger: Optional[logging.Logger] = None,
     ):
         """Initialize a ``VotingLearner`` object with the specified settings."""
         # initialize various attributes
         self._model = None
         self.voting = voting
-        self.label_dict = None
+        self.label_dict: Dict[LabelType, int] = {}
         self.logger = logger if logger else logging.getLogger(__name__)
```

Comment:
no type definition in the rest of the fields  (sorry)

Reply:
Don't need them in the rest of the fields since `mypy` can infer those. It only complained about this one.

Reply:
oh no I'm getting worse than mypy.